fearless/Common/View/TitleValueView.swift

```diff
@@ -0,0 +1,56 @@
+import UIKit
+import SoraUI
+
+final class TitleValueView: UIView {
+    let titleLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = R.color.colorLightGray()
+        label.font = UIFont.p1Paragraph
+        return label
+    }()
+
+    let valueLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = R.color.colorWhite()
+        label.font = UIFont.p1Paragraph
+        return label
+    }()
+
+    let borderView: BorderedContainerView = {
+        let view = BorderedContainerView()
+        view.backgroundColor = .clear
+        view.borderType = .bottom
+        view.strokeWidth = 1.0
+        view.strokeColor = R.color.colorDarkGray()!
+        return view
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupLayout()
+    }
+
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupLayout() {
+        addSubview(borderView)
+        borderView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+
+        addSubview(titleLabel)
+        titleLabel.snp.makeConstraints { make in
+            make.leading.centerY.equalToSuperview()
+        }
+
+        addSubview(valueLabel)
+        valueLabel.snp.makeConstraints { make in
+            make.trailing.centerY.equalToSuperview()
+            make.leading.greaterThanOrEqualTo(titleLabel.snp.trailing).offset(8.0)
```

Comment:
I suggest to move all insets to local Constants as we do in other views

Reply:
So we are moving to code based layout and suggestion here is that we don't move constants to the header because it is used in one place with clear naming